### ISSUE TITLE
searching for tags not showing the pages with the containing tags

### ISSUE THREAD

**Issue by steffenri** [plone/volto#845]
Plone 5.2 / Python 3 / Volto master branch

Steps to reproduce:
1. checkout volto master branch latest
2. create a page and add a tag to it
3. search for the tag in the search bar
-> expected behavior: searching should show all pages with the tags you searched for
-> actual behavior: search not working, not showing anything at all
-> if you click the tags on the pages directly, its not showing anything either
![searching-in-searchbar](https://user-images.githubusercontent.com/54619639/63927526-2a73c200-ca4e-11e9-9a5a-5a0d8b1283b6.png)

**Comment by csenger** [comment]
This happens cause the Volto search view does not handle the missing SearchableText parameter correctly and uses `SeachableText=` or `SearchableText=null*` (4.0.0alpha) in the query string of the `@search` request. The api endpoint does not return any results in this case.

The \@search request for https://volto.kitconcept.com/search?Subject=hamburger is

```
curl 'https://volto.kitconcept.com/api/@search?SearchableText=&Subject=hamburger&path=' \
  -H 'accept: application/json' \
  -H 'cache-control: no-cache'
```

If you remove `SearchableText=` from the request it works
```
curl 'https://volto.kitconcept.com/api/@search?Subject=hamburger&path=' \
  -H 'accept: application/json' 
  -H 'cache-control: no-cache' 
```

@sneridagh Do you think we should fix the \@search endpoint to ignore `SearchableText=&...` in the query?

**Comment by sneridagh** [comment]
@carsten indeed,

https://github.com/plone/volto/blob/master/src/components/theme/Search/Search.jsx#L68

Should ignore it if bot present (like the other ones).

**Comment by csenger** [comment]
Fixed by #865. Will be release in 4.0.0-alpha.2
